frontend/components/utils/AnimatedNumber.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <animated-number
+        v-if="value"
+        :value="value.toFixed(decimalPlaces)"
+        :format-value="formatValue"
```

Comment:
`formatValue` is used for the format of the "transition" numbers. This is to make sure that when it counts up from 0 to 100 the numbers correspond with the `formatValue` style. 


> * provide value as string

Do you want this component to only accept strings or should I just replace `.toFixed` with `.toString()`?

Reply:
If this component (`animated-number-vue`) accepts (and uses) `formatValue` function that expects number/bignumber, then where does it get number/bignumber if you only give it a string?

> Do you want this component to only accept strings or should I just replace `.toFixed` with `.toString()`?

What's wrong with providing value as number as suggested above?

Reply:
> What's wrong with providing value as `number` without any conversion as suggested above? Since the conversion should happen via `formatValue`

It wouldn't be an issue with just numbers, however we run into the issue that the component does not support BigNumbers. Meaning I can:

a.) check if the value is a BigNumber, if yes add `.toNumber()` (this function however does not work with normal numbers or string)
b.) convert numbers and BigNumbers to strings in order to not have to check if it is a BigNumber as both BigNumbers and numbers can be converted to string with either `.toString()` or `.toFixed()`.

Reply:
Ok, then, this component somehow converts this value back into number in order to pass to the `fromatValue` again 😅 

Please go with `b.)` but please use the same `formatValue` method for conversion

Reply:
Sounds good!